content/blogs/newsletter-101.md

```diff
@@ -0,0 +1,69 @@
+---
+title:       "Newsletter 101 - July 2021 news"
+author:      "Extravert34"
+date:        2021-07-26
+tags:        [ newsletter ]
+---
+
+Hello ReactOS followers! This report covers changes in the project during February-July 2021.
+And we definitely have some things to highlight!
+
+## amd64 build is getting more stable
+Timo Kreuzer (tkreuzer) worked hard on various parts of the kernel and HAL, fixing issues here and there.
+Structured Exception Handling (SEH) support for the amd64 architecture was finished, various bugs around the kernel are fixed.
+A major issue with interrupt handling in HAL was also fixed in May, which finally allowed a semi-stable boot in a virtual environment.
+
+## i686 APIC HAL now works
+ReactOS historically uses a legacy HAL on i686 architecture, which works with good old Intel 8259 PIC's interfaces for interrupt handling.
+This works in a single CPU environment, but for multiprocessor support, a more modern APIC interrupt handling is required.
+Timo Kreuzer also did a great job here reviving old code for APIC support.
+APIC HAL can be installed now by choosing the corresponding menu entry in the installer.
```

Comment:
Wasn't there Justin Miller part of this?

Reply:
A lot what I did there was super seeded by a much better implementation, so I never really submitted it for review or anything as there wasn't a point. I've learned a whole bunch more sense then too and in hindsight it really wasn't that great.
However, the MADT/MPS tables are a thing that'll be part of it eventually be part of all the APIC HALs but that isn't part of master right now. Sooner then later it will be though.